src/Apps/Artwork/Components/ArtworkDetails/index.tsx

```diff
@@ -44,15 +45,16 @@ export const ArtworkDetails: React.SFC<ArtworkDetailsProps> = props => {
       <Tabs>
         {renderAbout && (
           <Tab name="About the work">
-            <AboutTheWork artwork={artwork as any} />
-            <Checklist artwork={artwork as any} />
-            <AdditionalInfo artwork={artwork as any} />
+            <AboutTheWorkFromArtsy artwork={artwork} />
```

Comment:
Had to split this artwork details into 2 parts here because the details from partner supposed to be in a StackableBox now and I needed that box to be on a same level as other StackableBoxes below.